setup.cfg

```
@@ -1,2 +1,9 @@
 [aliases]
 test=pytest
+
+[flake8]
+statistics = True
+max-line-length = 80
+ignore = E501, B008, B011, W503
+select = C,E,F,W,B,B9
+exclude = docs,.svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg
```

Newline at end of file.
```suggestion
exclude = docs,.svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg

```